app/assets/src/components/layout/narrow_container.scss

```diff
@@ -1,7 +1,7 @@
 .narrowContainer {
-  max-width: 1280px;
+  max-width: 1308px;
   margin: 0 auto;
-  width: 100%;
+  width: calc(100% - 28px);
```

Comment:
please add a comment to explain the `-28px`

Reply:
done